src/Lender.sol

```diff
@@ -548,20 +519,20 @@ contract Lender {
         // Fetch the principal token for this lend call
         address pt = IMarketPlace(marketPlace).markets(u, m, p);
 
-        // Get the starting balance for the Lender
-        uint256 starting = IERC20(pt).balanceOf(address(this));
-
         // Conduct the lend operation to acquire principal tokens
-        (bool success, ) = adapter.delegatecall(
+        (bool success, bytes memory returndata) = adapter.delegatecall(
```

Comment:
I do like utilizing returns if available -- That said the last time around we did the pt math just to be _safe_. We confident this is no longer necessary / that we're testing the returns sufficiently?